code/ship/ship.cpp

```diff
@@ -11892,11 +11892,6 @@ int ship_fire_primary(object * obj, int stream_weapons, int force, bool rollback
 								{
 									flak_set_range(&Objects[weapon_objnum], flak_range - winfo_p->untargeted_flak_range_penalty);
 								}
-
-								if ((winfo_p->muzzle_flash>=0) && (((shipp==Player_ship) && (vm_vec_mag(&Player_obj->phys_info.vel)>=45)) || (shipp!=Player_ship)))
```

Comment:
Doesn't part of this check keep the player from having their view obstructed by a muzzle flash if they are moving fairly slowly?  Doesn't that need to be preserved somewhere else?

Reply:
Yes, this was an already awkward way to handle that, but normal muzzle flash logic knows how to deal with that, as it depends on several factors, like whether 'show ship model' or '$render player muzzle flashes" is used.

Reply:
Ok.  Sounds good then.